<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QWidget" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <widget class="my_Label" name="bag_label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>690</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>背包</string>
   </property>
  </widget>
  <widget class="my_Label" name="set_label">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>690</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="my_Label" name="other_client_label">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>690</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dbg_mode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>93</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="my_Label" name="game_label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>690</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>战斗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dbg_exp">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dbg_fight">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>550</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dbg_get_pokemon">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="exp_label">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>620</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> 分配经验值</string>
   </property>
  </widget>
  <widget class="QLabel" name="fight_label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>620</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  战斗</string>
   </property>
  </widget>
  <widget class="QLabel" name="get_pokemon_label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>620</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>获取测试精灵</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="dbg_log">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>241</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dbg_exit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>退出测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="bd1">
   <property name="geometry">
    <rect>
     <x>801</x>
     <y>64</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bd2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>230</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_Label</class>
   <extends>QLabel</extends>
   <header location="global">my_Label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
